### ISSUE TITLE
is this project still maintained? last relase is 3 years old

### ISSUE THREAD

**Issue by travelg** [intika/Librefox#141]
is this project still maintained? last release is 3 years old
Thanks

**Comment by travelg** [comment]
> relase

**Comment by StoppingBuck** [comment]
@travelg Nope, it's dead. The community-maintained active fork is https://librewolf-community.gitlab.io/

**Comment by ioc2e3** [comment]
> ... . The community-maintained active fork is https://librewolf-community.gitlab.io/

Got it ...

.
